asyncpg/connection.py

```diff
@@ -730,3 +772,20 @@ def _create_future(loop):
         return asyncio.Future(loop=loop)
     else:
         return create_future()
+
+
+def _detect_server_capabilities(server_version, connection_settings):
+    if hasattr(connection_settings, 'crdb_version'):
+        # CocroachDB detected.
+        advisory_locks = cursors = notifications = plpgsql = sql_reset = False
```

Comment:
Can you make it a multiline assignment. Will minimize the diff when we need to add a new one.